--- yt_dlp/utils/_utils.py
+++ yt_dlp/utils/_utils.py
@@ -640,21 +640,20 @@
     def replace_insane(char):
         if restricted and char in ACCENT_CHARS:
             return ACCENT_CHARS[char]
-        elif not restricted and char == '\n':
-            return '\0 '
-        elif is_id is NO_DEFAULT and not restricted and char in '"*:<>?|/\\':
-            # Replace with their full-width unicode counterparts
-            return {'/': '\u29F8', '\\': '\u29f9'}.get(char, chr(ord(char) + 0xfee0))
-        elif char == '?' or ord(char) < 32 or ord(char) == 127:
-            return ''
+        if char == '?' or ord(char) < 32 or ord(char) == 127:
+           return ''
         elif char == '"':
             return '' if restricted else '\''
         elif char == ':':
-            return '\0_\0-' if restricted else '\0 \0-'
+            return '_-' if restricted else ' -'
         elif char in '\\/|*<>':
-            return '\0_'
-        if restricted and (char in '!&\'()[]{}$;`^,#' or char.isspace() or ord(char) > 127):
-            return '' if unicodedata.category(char)[0] in 'CM' else '\0_'
+            return ' '
+        elif char[0] == '#':
+            return ''
+        if restricted and (char in '!&\'()[]{}$;`^,#' or char.isspace()):
+            return ' '
+        if restricted and ord(char) > 127:
+            return char
         return char
 
     # Replace look-alike Unicode glyphs
